imu_integration_node:
  ros__parameters:
    odom_sigma_velocity: 0.001
    odom_sigma_angular: 0.01
    position_sigma: 0.1
    velocity_sigma: 0.1
    theta_sigma: 0.006
    ba_sigma: 1.0
    bg_sigma: 0.00001
    gravity_sigma: 0.000001
    vi_sigma: 0.001
    thetai_sigma: 0.001
    ai_sigma: 0.001
    wi_sigma: 0.001
    update_jocabian: true
    use_acc: true
    integration_type: 0
    